external-crates/move/move-compiler/src/hlir/visitor.rs

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) The Move Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::command_line::compiler::Visitor;
+use crate::hlir::ast as H;
+use crate::shared::CompilationEnv;
+
+pub type HlirVisitorObj = Box<dyn HlirVisitor>;
+
+pub trait HlirVisitor {
```

Comment:
I think all of the letters of HLIR should be capitalized? But I'm the wrong person to ask. 

The name stands for "high level intermediate representation", which originally separated it from the "Move IR" (which in it's non-bytecode flavor is quite similar). 
Given that context, name this as you will :)

Reply:
No longer relevant at this visitor has been removed, but more generally, I was wondering this myself, but ultimately went for the coding convention (camel case).